lib/std/pdb.zig

```diff
@@ -506,15 +508,15 @@ const Msf = struct {
 
         // Sanity checks
         if (!mem.eql(u8, &superblock.FileMagic, SuperBlock.file_magic))
-            return error.InvalidDebugInfo;
+            return BaseError.InvalidDebugInfo;
```

Comment:
I fixed usage of the global error set in pdb and dwarf (there are similar edits in many places).

Reply:
`error.InvalidDebugInfo` and `BaseError.InvalidDebugInfo` are semantically identical and using `error.*` is currently the preferred way.